storage/innobase/log/log0log.cc

```diff
@@ -918,6 +924,9 @@ void log_t::persist(lsn_t lsn) noexcept
   ut_ad(!flush_lock.is_owner());
   ut_ad(latch_have_wr());
 
+  if (arch_sys)
+    arch_sys->log_sys()->wait_archiver(lsn);
+
   lsn_t old= flushed_to_disk_lsn.load(std::memory_order_relaxed);
```

Comment:
There seems to be a race condition here, because `Arch_Log_Sys::wait_archiver()` may release `log_sys.latch`. I think that it would be easiest to just not support the clone function with memory-mapped log writes (PMEM). An easy way to do this might be to have `log_mmap()` return `MAP_FAILED` when the clone plugin is loaded.